frontend/views/containers/contributions/InputAmount.vue

```diff
@@ -11,7 +11,7 @@ label.gi-fieldGroup
     span.control
       input.input(
         type='number'
-        :class="{'error': error }"
+        :class='{ error }'
```

Comment:
I am really liking this new rule you added 😄

Reply:
This new rule only demands the use of `'` instead of `"`.
The object shorthand is a JS feature already known :D